docs/docs/guides/web3_utils_migration_guide.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 6
+sidebar_label: Web3.utils
+---
+
+# Web3 Utils Migration Guide
+
+## Import
+
+To make use you only import the utility functions which are needed by your app we encourage the named import for `web3-utils` package. If you still want to import the full utils you can do it via. This change has no impact on the using the namespace `Web3.utils` or `web3.utils`.
```

Comment:
```suggestion
To make use you only import the utility functions which are needed by your app we encourage the named import for `web3-utils` package. This change has no impact on the using the namespace `Web3.utils` or `web3.utils`. If you still want to import the full utils it can be done as following : 
```